src/app/login-page/components/login-page/login-page.component.scss

```diff
@@ -1,20 +1,352 @@
+    border-radius: 10px;
+    border: solid 1px #cecece;
+    background-color: #ffffff !important;
+}
+.submit_button button{
+    width: 149px;
+    margin: 20px 100px 17px 101px;
+    padding: 16px 19px 14px 24px;
+    border-radius: 30px;
+    background-color: #5f38fa;
+    color: #ffffff;
+    border: solid 2px #ffffff;
+}
+.passwort_vergessen {
+    margin: 0 0px 0 110px;
+}
+.line {
+    height: 1px;
+    margin: 40px 70px 20px 70px;
+    background-color: #fff;
+}
+.login_form_buttom_text{
+    text-align: center;
+    color: #ffffff;
+    padding: 0 0 25px 0;
+    width: 64%;
+    margin: 0 auto;
+}
+span.tm {
+    font-size: 8px;
+    padding: 0 10px 0px 0;
+    vertical-align: text-top;
+}
+.detektiv_werden_btn{
+    color: #fff;
+    font-weight: bold;
+    width: 42%;
+    margin: 0 auto;
+    padding: 12px 0px 12px 20px;
+    border-radius: 30px;
+    border: solid 2px #ffffff;
+    background: transparent;
+}
+.detektiv_werden_btn button{
+    background: transparent;
+    border: unset;
+    color: #fff;
+}
+ 
+.second_warpper {
+    padding-top: 80px;
+    width: 75%;
+    margin: 0 auto;
+    padding-bottom: 100px;
+}
+.die_letzten_section {
+    width: 50%;
+    float: left;
+    position: relative;
+    left: 22%;
+}
+.unsere_partner_section {
+    width: 40%;
+    float: left;
+    padding-left: 60px;
+}
+.second_warpper:after {content: '';clear: both;display: block;}
+.die_letzten_section h3 {
+    padding-left: 20px;
+}
 
-	.logo-section {
-		margin-bottom: 50px;
-	}
+ul.records {
+    padding-left: unset;
+}
+.records .first_record{
+    list-style: none;
+    display: flex;
+}
+.records .first_record p {
+    margin: unset;
+}
+.records .first_record p.fd {
+    font-size: 25px;
+    font-weight: bold;
+    color: #0daf00;
+    padding: 10px 38px 10px 0;
+}
+.records .first_record p.phone {
+    font-weight: bold;
+}
+.records .first_record .phone .f_ip {
+    font-weight: normal;
+}
+.records .first_record p.s_ip {
+    padding-top: 22px;
+}
+.records .first_record p.ld {
+    padding: 22px;
+}
 
-	.login-components-container {
-		display: flex;
-		flex-direction: column;
-		align-items: center;
-	}
+.records .second_record{
+    list-style: none;
+    display: flex;
+}
+.records .second_record p {
+    margin: unset;
+}
+.records .second_record p.fd {
+    font-size: 25px;
+    font-weight: bold;
+    color: #d10e00;
+    padding: 10px 38px 10px 0;
+}
+.records .second_record p.phone {
+    font-weight: bold;
+}
+.records .second_record .phone .f_ip {
+    font-weight: normal;
+}
+.records .second_record p.s_ip {
+    padding-top: 22px;
+    padding-left: 15px;
+}
+.records .second_record p.ld {
+    padding: 22px;
+}
+
+
+.records .third_record{
+    list-style: none;
+    display: flex;
+}
+.records .third_record p {
+    margin: unset;
+}
+.records .third_record p.fd {
+    font-size: 25px;
+    font-weight: bold;
+    color: #d10e00;
+    padding: 10px 38px 10px 0;
+}
+.records .third_record p.phone {
```

Comment:
instead  of giving classes, for "first" "second" "third"
Can  you use general css rules?

We should make this list into its own separate component, with separate folder with
html, scss, spec.ts,and .ts file,   like the other components that already exist.

All the scss styles should be in there